test/src/test/java/org/corfudb/integration/AbstractIT.java

```diff
@@ -96,19 +101,66 @@ public AbstractIT() {
      */
     @Before
     public void setUp() throws Exception {
+        // these tests kill corfu server or lr server in the middle, so need to kill them immediately
```

Comment:
I think this comment is not clear and can be reworded

Reply:
Done, Thank you.